core/src/main/java/com/afollestad/materialdialogs/utils/MDUtil.kt

```diff
@@ -234,6 +234,13 @@ object MDUtil {
     }
   }
 
+  @RestrictTo(LIBRARY_GROUP) fun TextView?.maybeSetAdditionalPadding(height: Int): Int {
+    if (this == null) return 0
+    val fm = paint.fontMetrics
+    val measuredHeight = fm.descent - fm.ascent
```

Comment:
It might be better to call `measuredHeight` here `textHeight` or something, since `measuredHeight` is used in many other places in the context of the view itself